### ISSUE TITLE
Using L13: Modbus-Verbindung Bedieneinheiten?

### ISSUE THREAD

**Issue by spamtraxBeta** [ThomasRgbg/esp8266-pichler#1]
Hello Thomas,

I've got a Pichler LG350, too.

Just found this repository which looks quite interesting.

May I ask where you connected the MAX485 to the LG350 which has several Modbus interfaces?

For me "L13: Modbus-Verbindung Bedieneinheiten" seems to be the most likely one because the others do not make much sense to me for requesting information/adjusting ventialtion level...

(See page 53 in https://www.pichlerluft.at/pichler-kompaktlueftungsgeraete.html?file=files/content/downloads/Komfortlueftung/Betriebsanleitungen/08Betriebsanleitung_LG350-LG450.pdf)

- L11: Externer Modbus für GLT
- L13: Modbus-Verbindung Bedieneinheiten
- L16: Modbus-Verbindung Ventilatoren
- L17: Modbus-Verbindung externe Sensorik (CO2 / r.F.) (Max. 2 Sensoren)

And just to be sure, you connected 
- A/B from the MAX with corresponding A/B from the Lx port 
- VCC and GND from the MAX with 3.3 and GND from ESP8266 ? (according https://www.analog.com/media/en/technical-documentation/data-sheets/MAX1487-MAX491.pdf, page 7 it needs Positive Supply: 4.75V ≤ VCC ≤ 5.25V, not 3.3 but seems to work anyway...)

Thanks a lot in advance,
Martin

**Comment by ThomasRgbg** [comment]
Hi Martin, 

I use the L11 GLT (Gebäudeleittechnik, which is the interface for remote controlling the unit). The "Bedieneinheit" just stays as is and works in parallel fine. (Whatever comes last will be used, remote vs. button pressed)

For Modbus / RS485 you only need A/B, since it is a differential interface, whereas GND is more for shielding, incase you use a longer cable. With the VCC I have no experience, but I guess providing the supply for an ESP should be good. 

Sorry for not having more specific information, the whole code here was more for a temporary solution. Meanwhile I have an Raspberry PI (with a boring USB-RS485 adapter) instead, since it now controls multiple other devices as well. (But its basically about the same code for the Pichler control) 

Thomas

**Comment by ThomasRgbg** [comment]
I have no idea how to do here a private message, maybe you can drop me a mail at tom punkt gmx de (remove the first space, add the "@" and ".")  :)

**Comment by spamtraxBeta** [comment]
Hi Thomas,

you saved my day, many thanks (also from my wife :-D )

I connected to L11 now and all the CRC errors are gone.
(Before I connected to L13 in parallel to the "Bedieneinheit" (operating panel?)... yeah... stupid me, two masters on the bus won't do...)

Btw. I think the register list is not confidential.

There's this page:
https://www.pichlerluft.at/pichler-kompaktlueftungsgeraete.html

When you scroll down there are tons of documents.
There's also a chapter "Modbusliste" which provides this document describing the modbus settings and data points:

[LIST_Modbus_ES2020_FW_Cv14_LG350_LG450_LG740_LG1000_v14_D-E.xlsx](https://www.pichlerluft.at/pichler-kompaktlueftungsgeraete.html?file=files/content/downloads/LOGIN/LG%20350_LG%20450_LG%20740_LG%201000%20SK/LIST_Modbus_ES2020_FW_Cv14_LG350_LG450_LG740_LG1000_v14_D-E.xlsx)

Best regards,
Martin

PS: I also failed to send a private message, that's why I raised this issue. Still responded here, maybe it's useful to someone else, too.
If there's still need for PM: martin_wvft7xsc@ma-register.ertl-dstf.de
